cmd/notation/login.go

```diff
@@ -126,3 +146,32 @@ func readLine() (string, error) {
 	password = strings.TrimSuffix(password, "\r")
 	return password, nil
 }
+
+func readUsernameFromPrompt(reader *bufio.Reader) (string, error) {
+	fmt.Print("Username: ")
+	username, err := reader.ReadString('\n')
+	if err != nil {
+		return "", fmt.Errorf("error reading username: %w", err)
+	}
+	username = strings.TrimSpace(username)
+	return username, nil
+}
+
+func readPasswordFromPrompt(reader *bufio.Reader) (string, error) {
+	fmt.Print("Password: ")
+	if term.IsTerminal(syscall.Stdin) {
```

Comment:
cmd\notation\login.go:162:21: cannot use syscall.Stdin (variable of type syscall.Handle) as int value in argument to term.IsTerminal
cmd\notation\login.go:163:42: cannot use syscall.Stdin (variable of type syscall.Handle) as int value in argument to term.ReadPassword

I tried to compile it on Windows, but it didn't work. We may need to convert `syscall.Stdin` to be an `int` type value

Reply:
Thanks. Tried the latest change in my windows desktop and it can build now.